infrastructure-setup/scripts/configuration-sample.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# We assume that all ods repositories are placed next to each other
+# By default it is one level up from this script
+
+ODS_DIR=""
+ODS_SAMPLE_DIR=`realpath ${ODS_DIR}/ods-configuration-sample`
+ODS_CONFIG_DIR=`realpath ${ODS_DIR}/ods-configuration`
+
+# check if we have a ods-configuration
+# and create one
+if [[ ! -d "${ODS_CONFIG_DIR}" ]] ; then
+  echo "creating ods configuration directory"
+  mkdir ${ODS_CONFIG_DIR}
+ fi
+ if [ "$(ls -A ${ODS_CONFIG_DIR})" ]; then
```

Comment:
Indentation looks off. Also, why do we execute this even when we just created the dir? Or am I reading it wrong? :)

Reply:
Oh now I get it, your checking if it is empty. Well then just indentation I guess ...